edn/src/types.rs

```diff
@@ -563,7 +563,7 @@ pub trait FromMicros {
 
 impl FromMicros for DateTime<UTC> {
     fn from_micros(ts: i64) -> Self {
-        UTC.timestamp(ts / 100_000, ((ts % 100_000).abs() as u32) * 1_000)
+        UTC.timestamp(ts / 1_000_000, ((ts % 1_000_000).abs() as u32) * 1_000)
```

Comment:
Probably not — my reasoning is: we're likely to wrap `DateTime` rather than directly implement `FromMicros`, and if we _do_ do it again we'll probably forget to use the constant!

hashtag-realism hashtag-bugs